canonical-websites/www.ubuntu.com
# 1616 cloud public cloud

## Done

* update /cloud/public-cloud to vbt

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/cloud/public-cloud)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/2.%20Cloud?preview=cloud-public-cloud.png)

## Issue / Card

Fixes #1616
